### ISSUE TITLE
h1 missing standalone-resource__type-of-guidance

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-minimal-header-design#98]
missing in supplemental guidance and rules

for example "Cognitive Accessibility Design Pattern" missing from :
![image](https://user-images.githubusercontent.com/6991814/169567522-0375d0c6-cadb-424f-b2a8-01ffa0af0308.png)

**Comment by shawna-slh** [comment]
p.s. @SteveALee I need to talk with you briefly Friday. please ping me in IRC.

**Comment by shawna-slh** [comment]
@SteveALee here's the issue for the missing "Cognitive Accessibility Design Pattern" etc. that I mentioned earlier today.

**Comment by SteveALee** [comment]
Fixed and published - https://github.com/w3c/wai-website/commit/fe1e8b6bb021da3d1997149ee25abf9d4a782878
